src/coreComponents/virtualElement/unitTests/testVirtualElementBase.cpp

```diff
@@ -131,6 +151,76 @@ checkStabilizationMatrixConsistency
   }
 }
 
+template< localIndex MAXCELLNODES, localIndex MAXFACENODES >
+static void checkSumOfQuadratureWeights( real64 const & cellVolume )
+{
+  using VEM = ConformingVirtualElementOrder1< MAXCELLNODES, MAXFACENODES >;
+  real64 sum = 0.0;
+  for( localIndex q = 0; q <
+       VEM::getNumQuadraturePoints(); ++q )
```

Comment:
Either break each statement on its own line, or keep then all on the same line
```suggestion
  for( localIndex q = 0; q < VEM::getNumQuadraturePoints(); ++q )
```